scripts/changelog.ts

```diff
@@ -77,6 +77,7 @@ export default async function run(
     '@spartacus/incubator': 'projects/incubator',
     '@spartacus/cds': 'projects/cds',
     '@spartacus/my-account': 'feature-libs/my-account',
+    '@spartacus/product': 'feature-libs/product',
```

Comment:
We also need to add the lib at the enf od this file, when we detect the library by name.
Missed that for my-account as well.